source/js/front/form-validate.js

```diff
@@ -0,0 +1,54 @@
+const eventFormValidate = {
+    setupFormValidate: () => {
+        const forms = document.querySelectorAll('.js-event-form');
+        forms.forEach(form => {
+            const inputs = form.querySelectorAll('input, textarea, select');
+
+            // Validate fields on change
+            ['keyup', 'change'].forEach(function(e) {
+                inputs.forEach(input => {
+                    input.addEventListener(e, function() { eventFormValidate.validateInput(input); });
```

Comment:
couldn't you just simplify this to an arrow function?
```suggestion
                    input.addEventListener(e, () => eventFormValidate.validateInput(input));
```